upup/pkg/fi/cloudup/scalewaytasks/loadbalancer.go

```diff
@@ -191,3 +194,38 @@ func (l *LoadBalancer) RenderScw(t *scaleway.ScwAPITarget, actual, expected, cha
 
 	return nil
 }
+
+type terraformLBIP struct{}
+
+type terraformLoadBalancer struct {
+	Type string                   `cty:"type"`
+	Name *string                  `cty:"name"`
+	Tags []string                 `cty:"tags"`
+	IPID *terraformWriter.Literal `cty:"ip_id"`
+}
+
+func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, actual, expected, changes *LoadBalancer) error {
+	tfName := strings.Replace(fi.ValueOf(expected.Name), ".", "-", -1)
+
+	tfLBIP := terraformLBIP{}
+	err := t.RenderResource("scaleway_lb_ip", tfName, tfLBIP)
+	if err != nil {
+		return err
+	}
+
+	tfLB := terraformLoadBalancer{
+		Type: "LB-S",
+		Name: expected.Name,
+		Tags: expected.Tags,
+		IPID: tfLBIP.TerraformLink(tfName),
```

Comment:
Similar to instance IP.
```suggestion
		IPID: terraformWriter.LiteralProperty("scaleway_lb_ip", tfName, "id")
```